File: ios/CSChat/Classes/Domain/Model/ConversationResponse.swift
//
//  ConversationResponse.swift
//  CSChat
//
//  Created by Hugo Lefrancois on 2017-03-24.
//  Copyright © 2017 Mirego. All rights reserved.
//

import UIKit
import ObjectMapper

class ConversationResponse: Mappable
{
    var conversations: [ConversationSummary]? = nil
    var users: [User]? = nil

    required init?(map: Map)
    {
    }

    func mapping(map: Map)
    {
        conversations <- map["conversations"]
        users <- map["users"]
    }
}

File: android/app/src/main/java/com/mirego/cschat/models/ConversationResponse.java
package com.mirego.cschat.models;

import java.util.List;

public class ConversationResponse {

    private List<Conversation> conversations;

    private List<User> users;

    public List<Conversation> getConversations() {
        return conversations;
    }

    public void setConversations(List<Conversation> conversations) {
        this.conversations = conversations;
    }

    public List<User> getUsers() {
        return users;
    }

    public void setUsers(List<User> users) {
        this.users = users;
    }
}
